dart/lib/src/builder.dart

```diff
@@ -516,34 +505,42 @@ class Builder {
 }
 
 class _StackValue {
-  Object _value;
-  int _offset;
+  late Object _value;
+  int? _offset;
   ValueType _type;
   BitWidth _width;
-  _StackValue.WithNull() {
-    _type = ValueType.Null;
-    _width = BitWidth.width8;
-  }
-  _StackValue.WithInt(int value) {
-    _type = value != null ? ValueType.Int : ValueType.Null;
-    _width = BitWidthUtil.width(value);
-    _value = value;
-  }
-  _StackValue.WithBool(bool value) {
-    _type = value != null ? ValueType.Bool : ValueType.Null;
-    _width = BitWidth.width8;
-    _value = value;
-  }
-  _StackValue.WithDouble(double value) {
-    _type = value != null ? ValueType.Float : ValueType.Null;
-    _width = BitWidthUtil.width(value);
-    _value = value;
-  }
-  _StackValue.WithOffset(int value, ValueType type, BitWidth width) {
-    _offset = value;
-    _type = type;
-    _width = width;
-  }
+
+  _StackValue.WithNull()
```

Comment:
The formatting changes are pretty strange here - I don't think this is even how dartfmt would do it, right?

Reply:
indeed, not sure where that come from. can I apply dartfmt -w instead?

Reply:
Go for it :)